tests/subscriptions/test_executor.py

```diff
@@ -35,23 +36,22 @@ def test(self):
                 resolution=timedelta(minutes=1),
             ),
         )
+
         now = datetime.utcnow()
-        task = ScheduledTask(now, subscription)
         tick = Tick(
             offsets=Interval(1, 2),
             timestamps=Interval(now - timedelta(minutes=1), now),
         )
 
-        future = executor.execute(task, tick)
-        result = future.result()
-
+        result = executor.execute(ScheduledTask(now, subscription), tick).result()
         assert result["data"][0]["count"] == 10
 
-        tick = Tick(
-            offsets=Interval(5000, 5001),
-            timestamps=Interval(now + timedelta(hours=10), now + timedelta(hours=11)),
-        )
-        future = executor.execute(task, tick)
-        result = future.result()
+        result = executor.execute(
+            ScheduledTask(
+                now + timedelta(minutes=self.minutes) + subscription.data.time_window,
```

Comment:
Executing a task "in the future" doesn't seem very natural to me here, but this is the way the test data is structured and I don't really want to change it right now.